app/notification-queue/server/NotificationQueue.ts

```diff
@@ -0,0 +1,232 @@
+import { Meteor } from 'meteor/meteor';
+import { Collection, ObjectId } from 'mongodb';
+
+import { NotificationQueue, Users } from '../../models/server/raw';
+import { sendEmailFromData } from '../../lib/server/functions/notifications/email';
+import { PushNotification } from '../../push-notifications/server';
+
+const UsersCollection: Collection = Users.col;
+
+interface INotificationItemPush {
+	type: 'push';
+	data: {
+		roomId: string;
```

Comment:
why is this information duplicated from `INotification`?

if really needed, why not use `rid` and `uid` as used there?

Reply:
I didn't change the content of the push and email, just saving them and reading.